src/planet.ts

```diff
@@ -80,7 +83,7 @@ class Planet {
     if (width != this.width || height != this.height) {
       this.mesh.destroy();
       (<THREE.Material>this.mesh.material).dispose();
-        this.createMeshes(width, height);
+        this.createMesh(width, height);
```

Comment:
It was called "createMeshes" before because it was a relic from an earlier PR where I used two separate meshes for the zoomed-out and zoomed-in surface. I may revisit that idea later, but for now let's keep it simple.